content/en/docs/contribute/style/hugo-shortcodes/index.md

```diff
@@ -243,4 +243,4 @@ Renders to:
 * Learn about [using page templates](/docs/home/contribute/page-templates/).
 * Learn about [staging your changes](/docs/home/contribute/stage-documentation-changes/)
 * Learn about [creating a pull request](/docs/home/contribute/create-pull-request/).
-{{% /capture %}}
```

Comment:
Sorry about this one :(

Reply:
This file/change is not needed (this file set to the original version)?